### PR TITLE
Nathi discourse

### PR DESCRIPTION
**Pull request by NathDrac**
Hi all

I have written A LOT, and my fingers and eyes are bleeding ;-) Please check, if this can be merged.

### CODE REVIEW

File: Thesis.tex [hunk 1]
```diff
@@ -178,9 +177,13 @@ \subsection{Open Source Software}
...
+
+Furthermore, however, the user is most often required to agree to software license agreements, stating further limitations to which the license of the software is subject to. This mechano is summarized in Wikipedia as following: "Proprietary software licenses often proclaim to give software publishers more control over the way their software is used by keeping ownership of each copy of software with the software publisher. By doing so, Section 117 does not apply to the end-user and the software publisher may then compel the end-user to accept all of the terms of the license agreement, many of which may be more restrictive than copyright law alone."\parencite{WikiSoftLic} These terms might e.g. limit the duration of the license to one year, prolonged only in case of further payment. Microsoft e.g. grants the rights to use Office 365 only for the duration of a year, after which a repeated payment is required\parencite{Off365}.
```
Comment (KilnOfTheSecondFlame) [inline Thesis.tex]: Either use "Furthermore" or "However" - Both together are unnecessary and cumbersome

Comment (KilnOfTheSecondFlame) [reply Thesis.tex]: 'mechano' is a colloquialism, beter use: "This mechanism is summarized [...]"

Comment (KilnOfTheSecondFlame) [inline Thesis.tex]: Section 117 of what? The Licencing Act 2003? This: Application for grant or renewal of personal licence ?
If so, please specify, which right Section 117 would grant an End-User, before explaining what not applying means.

File: Thesis.tex [hunk 2]
```diff
@@ -222,11 +225,85 @@ \subsection{Reasonable and non-discriminatory licensing}
...
+	\item In the current age, music is often not the creation of an inspired artist but of composers creating goods based on market research, e.g. the current music taste and characterizations of idols for pre-teen and teenage girls, which does lead to floods of one-hit wonders and good singers that get burnt out for the financial gain of industry giants. As such one-hit wonders generate big revenue streams for the publishers that have also controlled the "compositions" (and I use that word loosely) and only have to pay for the actual performance and minor wages for the singers and musicians, the market gets saturated by low quality music leaving less space for really talented artists and making it hard for them to get acceptable contracts, as the risk for the publishers might be higher and the part of the revenue going to the artist is (rightfully) higher, hurting the industry's bottom line. This, however, leads to good artists giving up or not being able to generate an income by touring, as their music is not known enough, basically being drowned by low-quality noise.
```
Comment (KilnOfTheSecondFlame) [inline Thesis.tex]: So what if the market is flooded with low quality music?
Or said differently: How does this relate to copyright?

File: Thesis.tex [hunk 3]
```diff
@@ -222,11 +225,85 @@ \subsection{Reasonable and non-discriminatory licensing}
...
...o many things already exist. And innovation is important for the advancement of human culture. The digital age would not have been realized the way it has been were it not for Google. Google on the other hand would not have succeeded were it for OSS. The costs of proprietary operating systems for server farms the size of Google's would have made the rise of the search engine impossible. Another reason that OSS is to be preferred to closed source software (called CSS from here on) as described in \ref{ssec:SLic} is the data that is created and managed by the software. If a software publisher goes out of business, the software will no longer be updated or maintained. Within two or three operating system lifecycles, it is no longer possible to run that software at all, thus making all the data created worthless, because it is no longer readable. While standards can weaken this effect, not every software adheres to a standard, so the binary file formats are useless. Imagine using a comfortable database application for managing your family photos, and you lose the photos, just because the software is no longer available to read the archive. Such a scenario is realistic, but must not be.
```
Comment (KilnOfTheSecondFlame) [inline Thesis.tex]: Google on the other hand would not have succeeded were it _not_ for OSS.

Comment (KilnOfTheSecondFlame) [inline Thesis.tex]: Such a scenario is realistic, but _should_ not be.
At least that's what I think you meant =)

File: Thesis.tex [hunk 4]
```diff
@@ -222,11 +225,85 @@ \subsection{Reasonable and non-discriminatory licensing}
...
+
+Furthermore, there's nothing new anymore, only new combinations of existing techniques. The electric car is just a combination of battery technics, electronic motors and computers with concepts of existing cars. Drones are merely more than a combination of wireless communication, well known aerodynamics, and so on. Smartphones are merely a mixture between computers and telephones, and so on. Unfortunately, the patent system does not really address that. While there is a notion that one can't patent combinations that would come to mind if persons of the same arts of science look at the same problem, this obstacle is rarely even considered when patent officers grant patents.
+
+The society should ask itself if it wants to cater to the needs of greedy corporations and believe their propaganda, or if not the need to solve problems would lead to the discoveries of things without the financial incentives that patents offer. Society should ask itself, where its priorities should be: in advancing to a better future or in deepening deep pockets.
```
Comment (KilnOfTheSecondFlame) [inline Thesis.tex:289]: Nice! 👍

**Review by KilnOfTheSecondFlame — CHANGES_REQUESTED** [review]
Oh no - bleeding eyes are not good 😢 
But could you address my points, so that we are able to merge?

File: Thesis.tex (hunk 3 above)
Comment (NathDrac) [reply Thesis.tex]: I changed it. but I really meant "dies darf nicht der fall sein" - habe aber deine Abschwächung akzeptiert, da es ja so offentsischtlich aktuell der Fall ist.

Comment (KilnOfTheSecondFlame) [reply Thesis.tex]: Jop, das habe ich eigentlich damit gemeint, aber gebe dir recht "das darf nicht der Fall sein"

**Review by KilnOfTheSecondFlame — APPROVED** [review]

**Pull request merged**